docs/setup.md

```diff
@@ -2,41 +2,75 @@
 
 This document outlines the setup process for the `Superposition Platform`.
 
-## Installation for linux/MacOS
+## Installation for Linux/MacOS
 
-- Install Rust using rustup
-    `curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
-- Install wasm-pack
-    `curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh`
-- Install docker
-- make sure you have dependencies like libpq, openssl, libiconv,etc installed
-- make sure `make` command is installed
-  - Linux
+- **Install** [rust](https://rustup.rs).
+
+- **Install** [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/)
+
+- **Install Docker**
+
+- **Installing Lib Dependencies & Tools :** <br>
+    `libpq`, `openssl`, `libiconv` are required for building the project. `diesel_cli` and `aws cli` is needed for dev.
+
+    For MacOS :
     ```bash
-    sudo apt-get install build-essential
+    brew install libpq openssl libiconv awscli
     ```
-  - MacOS
+    Don't forget to add libpq to `PATH`.
     ```bash
-    xcode-select --install
+    export PATH="/usr/local/opt/libpq/bin:$PATH"
```

Comment:
For ensuring `libpq` is picked up by `diesel` and/or `diesel_cli`, it's sufficient to set the `PQ_LIB_DIR` environment variable, setting `PATH` is not necessary:

```shell
export PQ_LIB_DIR="$(brew --prefix libpq)/lib"
```

Also, would suggest using `brew prefix` instead of using `/usr/local` so that the path generated handles architecture differences (Intel vs Apple Silicon).